### DISCUSSION TITLE
Request to delete my account

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#19989] [Questions] [answered]
Hi, i would like to delete my supabase account

**Comment by GaryAustin1** [comment, marked as answer]
You need to send in a request to support.

**Comment by ghost** [comment]
Where can I submit a support request?

**Reply by GaryAustin1** [reply]
Normally, the dashboard it the upper top help tab is the best way.
Otherwise you can email them at support@supabase.io
